das-dashboard/backend/shared/utils/das_cli_config.py

```
@@ -0,0 +1,67 @@
+    if remote_host is not None:
+        profile = web_config.user_profile
+        ssh_username = profile.get("profile_username", "root")
+        ssh_key = profile.get("profile_ssh_keypath") or DEFAULT_SSHKEY_CLONE_PATH
+
+        if not os.path.exists(ssh_key):
+            raise ValueError("SSH key is not configured. Set up your profile first.")
+
+        cmd.extend(["--remote", "--host", remote_host, "-u", ssh_username, "-k", ssh_key])
+
+    try:
+        subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)